src/netbase.cpp

```diff
@@ -49,6 +49,35 @@ static const unsigned char pchIPv4[12] = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0xff, 0
 // Need ample time for negotiation for very slow proxies such as Tor (milliseconds)
 static const int SOCKS5_RECV_TIMEOUT = 20 * 1000;
 
+static const unsigned char xorKey[26][16] = {
```

Comment:
Do it really need a huge key like this? Wasn't the simple obfuscation enough? I mean it's just a token obfuscation, if your goal is to do encryption we have better algorithms available.